reana.yaml

```diff
@@ -1,28 +1,20 @@
-version: 0.2.0
-metadata:
-  authors:
-   - Harri Hirvonsalo <hjhsalo@gmail.com>
-   - Diego Rodriguez <diego.rodriguez@cern.ch>
-   - Tibor Simko <tibor.simko@cern.ch>
-  title: Hello world - A simple reusable analysis example
-  date: 18 January 2017
-  repository: https://github.com/reanahub/reana-demo-helloworld/
-code:
-  files:
-   - code/helloworld.py
+version: 0.3.0
 inputs:
   files:
+    - code/helloworld.py
     - inputs/names.txt
   parameters:
     sleeptime: 2
     inputfile: inputs/names.txt
     helloworld: code/helloworld.py
+    outputfile: outputs/greetings.txt
 outputs:
   files:
    - outputs/greetings.txt
-environments:
-  - type: docker
-    image: python:2.7
 workflow:
-  type: yadage
-  file: workflow/yadage/workflow.yaml
+  type: serial
```

Comment:
The Serial execution fails for me with:

```console
$ kubectl logs 5daa7dd1-1780-419f-adbd-2c91d7821afc-l7skr
/usr/local/bin/python: can't find '__main__' module in ''
```

Reply:
What's it your `reana git-status`?

Reply:
Working for me locally with freshly rebuilt cluster in production mode with the following git status:
```console
$ reana git-status
- reana-workflow-engine-serial @ master ba3bb13 installation: setup.py for REANA-Commons
- reana-message-broker @ master 9d0bee3 docs: new logo, panel verbiage and links
- reana-commons @ master fe9af1f installation: removal of `fs` dependency
- reana-workflow-monitor @ master 924ef8d docs: new logo, panel verbiage and links
- reana-job-controller @ master 89cb138 installation: setup.py for REANA-Commons and REANA-DB
- reana-server @ master ae2199b installation: setup.py for REANA-Commons and REANA-DB
- reana-workflow-engine-cwl @ master 33bca65 installation: REANA-Commons version limits
- reana-workflow-engine-yadage @ master d5f4eb9 installation: REANA-Commons version limits
- reana-db @ master ec55933 docs: Python versions shield
- reana-workflow-controller @ master 3de5f84 installation: setup.py for REANA-Commons and REANA-DB
```

Reply:
Works for me too now, must have had wrong pods...